infrastructure/disk.tf

```diff
@@ -114,9 +114,51 @@ resource "aws_s3_bucket" "data_refinery_compendia_bucket" {
   }
 }
 
-resource "aws_s3_bucket_metric" "compendia_metrics" {
+resource "aws_s3_bucket_metric" "compendia_bucket_metrics" {
```

Comment:
```suggestion
# Passing the name attribute as EntireBucket enables request metrics for the bucket. 
resource "aws_s3_bucket_metric" "compendia_bucket_metrics" {
```

Reply:
is this clearer @kurtwheeler

Reply:
Maybe also a link to documentation explaining this if possible?